gstreamer-gl-sys/src/lib.rs

```diff
@@ -519,6 +519,24 @@ impl ::std::fmt::Debug for GstGLDisplayClass {
     }
 }
```

Comment:
Also `GST_GL_DISPLAY_EGL_NAME` should be hidden behind the feature

Otherwise looks good to me. Will review all the .gir file changes in the other .gir files later

Reply:
Fix this one and I'll merge it, looks all good apart from this @ceyusa :)